src/EnergyPlus/DaylightingManager.cc

```diff
@@ -566,75 +559,61 @@ void CalcDayltgCoefficients(EnergyPlusData &state)
         }
     }
 
-    //           -----------
-    // ---------- ZONE LOOP ----------
-    //           -----------
-
-    for (ZoneNum = 1; ZoneNum <= state.dataGlobal->NumOfZones; ++ZoneNum) {
-        // Skip zones that are not Daylighting:Detailed zones.
-        // TotalDaylRefPoints = 0 means zone has (1) no daylighting or
-        // (3) Daylighting:DElight
-        if (state.dataDaylightingData->ZoneDaylight(ZoneNum).TotalDaylRefPoints == 0 ||
-            state.dataDaylightingData->ZoneDaylight(ZoneNum).DaylightMethod != DataDaylighting::iDaylightingMethod::SplitFluxDaylighting)
-            continue;
-
-        // Skip zones with no exterior windows in the zone or in adjacent zone with which an interior window is shared
-        if (state.dataDaylightingData->ZoneDaylight(ZoneNum).NumOfDayltgExtWins == 0) continue;
-
-        CalcDayltgCoeffsRefMapPoints(state, ZoneNum);
-
-    } // End of zone loop, ZoneNum
+    CalcDayltgCoeffsRefMapPoints(state);
```

Comment:
Now there is no for conditionals, Hoping example files and unit tests prove this logic.